doc/Overview/Logging/LoggingOverview.md

```diff
@@ -1,108 +1,182 @@
+The library offers a number of extension methods to simplify the logging of messages for a specific log level. The following example shows how to log a message using the `LogInformation()` extension method:
 
 ```csharp
-private IHost Host { get; }
-
-public App()
+public class MyClass
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLogging()
-        .ConfigureLogging(logBuilder =>
-                {
-                    logBuilder
-                        .SetMinimumLevel(LogLevel.Debug)
-                        .XamlLogLevel(LogLevel.Information)
-                        .XamlLayoutLogLevel(LogLevel.Information);
-                })
-        .Build();
-    // ........ //
+    private readonly ILogger<MyClass> _logger;
+
+    public MyClass(ILogger<MyClass> logger)
+    {
+        _logger = logger;
+    }
+
+    public void MyMethod()
+    {
+        _logger.LogInformation("My message");
+    }
 }
 ```
 
-## Serilog
+## Log Levels
 
-```csharp
-private IHost Host { get; }
+Sometimes it is necessary to write information to the log that varies in its verbosity or severity. For this reason, there are six log levels available to delineate the severity of an entry.
+
+Use the following convention as a guide when logging messages:
+
+| Level | Description |
+|-------|-------------|
+| Trace | Used for parts of a method to capture a flow. |
+| Debug | Used for diagnostics information. |
+| Information | Used for general successful information. Generally the default minimum. |
+| Warning | Used for anything that can potentially cause application oddities. Automatically recoverable. |
+| Error | Used for anything that is fatal to the current operation but not to the whole process. Potentially recoverable. |
+| Critical | Used for anything that is forcing a shutdown to prevent data loss or corruption. Not recoverable. |
+
+Because these extension methods are scoped to a specific log level, the standard `Log()` method on the `ILogger<T>` interface does not need to be used.
+
+## Configuring Logging Output
+
+The `UseLogging()` extension method accepts an optional `Action<ILoggingBuilder>` parameter that can be used to configure the logging output.
+
+### Setting the Minimum Log Level
+
+The minimum log level can be set by calling the `SetMinimumLevel()` extension method on the `ILoggingBuilder` instance. If no LogLevel is specified, logging defaults to the `Information` level which may not always be desirable. The following example shows how to change the minimum log level to `Debug`:
 
-public App()
+```csharp
+protected override void OnLaunched(LaunchActivatedEventArgs e)
 {
-    Host = UnoHost
-        .CreateDefaultBuilder()
-        .UseLogging()
-        .UseSerilog()
-        .Build();
-    // ........ //
-}
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host.UseLogging(
+                builder => {
+                    builder.SetMinimumLevel(LogLevel.Debug);
```

Comment:
A useful example is to base the log level on the current environment. I think there's an example of this in the Todo and/or Commerce app